packages/entity-browser/.npmignore

```diff
@@ -3,3 +3,5 @@ node_modules
 tests
 coverage
 .eslintrc
+jest.config.js
```

Comment:
good you added this but this change is completely independent of everything else. > Separate commit
And entity-browser isn't the only package.
also we have a package template in the plop folder. you can add these files there as well